cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_common_config.rs

```diff
@@ -50,32 +50,48 @@ impl From<RewardsAccountParams<LegacyLaneId>> for BridgeReward {
 	}
 }
 
+/// An enum representing the different types of supported beneficiaries.
+#[derive(Clone, Debug, Decode, Encode, Eq, MaxEncodedLen, PartialEq, TypeInfo)]
+pub enum BridgeRewardBeneficiaries {
+	/// A local chain account.
+	LocalAccount(AccountId),
+	/// A beneficiary specified by a VersionedLocation.
+	Location(VersionedLocation),
```

Comment:
Nit: from what I understand from the snowbridge logic I think this can be sxclusively an asset hub location. So should we rename it to ?`AssetHubLocation`. It's not very important, just saying because I'm not sure if it's easy to rename it after this is deployed on westend bridge hub.

Reply:
ok, makes sense, renamed: https://github.com/paritytech/polkadot-sdk/pull/7609/commits/56dceaeb7115a0adffecc07a25d72e30a3866f98

this enum just changes extrinsic and transaction version, so it is ok for BHW